### PR TITLE
Feat/image optimization

### PR DESCRIPTION
**Pull request by adon-cloudcannon**
# Context

Ok - another somewhat gnarly PR... but better, I think? MOST of the code relates to the image optimization :)

[Link to Notion ticket](https://www.notion.so/cloudcannon/11ty-Starter-Template-7abb3ae1482c4960a0eabc50cd808503?pvs=4)

# Additional information

Followed this video: https://youtu.be/Dsd831CSazA

This code features:

- 11ty image optimization plugin, implemented as per video
- instances where 'img' tags are used have been replaced with shortcodes - note: background-images and some smaller images (like svg icons) have not been replaced - discussion point for future work?
- Moved 'fetch-theme-colors' from the .cloudcannon prebuild into an eleventy prebuild event - this means we can remove multiple instances throughout the codebase (e.g. in the npm scripts in package.json + the cloudcannon prebuild file) - now local or deployed build will run the script as part of the prebuild step.
- started npm audit fixes (before I realized I was still on this branch and then stopped!) - replaced 2 packages with moderate vulnerabilities (other vulnerabilities addressed in another PR).
- Replaced Lorem Ipsum with more relevant copy (ChatGPT).

# Testing (tester)

The reviewer should check on this branch:

- The code
- The component(s) in Bookshop browser - (the /components page when running locally - not the 'standalone' browser)
- The site in CloudCannon [Link here](https://wonderful-mallard.cloudvent.net/)

### CODE REVIEW

File: .cloudcannon/prebuild
Comment (tomrcc) [inline .cloudcannon/prebuild:1]: Does this actually need 'npm i' in the prebuild? 

Check it without, but I don't think we need to do this here.

File: .gitignore
Comment (tomrcc) [inline .gitignore:1]: Can remove _site_translated potentially - that's a Rosey thing so should maybe only be present in the .gitignore on the Rosey branch

File: package.json
Comment (tomrcc) [inline package.json:1]: Unsure the exact process for moving fetch-theme-color into the eleventy prebuild as mentioned in the PR notes, but does it still need to be in `npm start`? 

Same question for process-breakpoints

**Review by tomrcc — APPROVED** [review]
Couple quick questions again, but happy to merge

File: .cloudcannon/prebuild
Comment (adon-cloudcannon) [reply .cloudcannon/prebuild:1]: Checked, can confirm that it works without it... removing it

File: .gitignore
Comment (adon-cloudcannon) [reply .gitignore:1]: Done

File: package.json
Comment (adon-cloudcannon) [reply package.json:1]: Yep - my bad... those are indeed inside the Eleventy prebuild so can be removed...

**Pull request merged**